internal/graph/check.go

```diff
@@ -253,8 +264,9 @@ func intersection(ctx context.Context, concurrencyLimit uint32, handlers ...Chec
 				err = result.err
 				continue
 			}
+			dbReads += result.resp.ResolutionMetadata.DatabaseReads
 
-			if !result.resp.GetAllowed() {
+			if !result.resp.Allowed {
```

Comment:
For this case, do we need to set the result.resp.ResolutionMetadata.DatabaseReads to be dbReads?

Otherwise, the result will only contains the dbRead from the last goroutine that returns the result.